types/emoji-js/index.d.ts

```diff
@@ -0,0 +1,47 @@
+// Type definitions for emoji-js 3.4
+// Project: https://github.com/iamcal/js-emoji
+// Definitions by: Jason Di Benedetto <https://github.com/me>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
```

Comment:
Library appears to be an UMD module. You also need to define `export as namespace EmojiConvertor;`